src/library/dao/trackdao.h

```diff
@@ -40,7 +48,7 @@ class TrackDAO : public QObject, public virtual DAO, public virtual GlobalTrackC
     void finish();
 
     TrackId getTrackId(const QString& absoluteFilePath);
-    QList<TrackId> getTrackIds(const QList<QFileInfo>& files, bool addMissingTracks);
+    QList<TrackId> resolveTrackIds(const QList<QFileInfo>& files, ResolveTrackIdOptions options);
```

Comment:
Should we add the default option = ResolveTrackIdOption::ResolveOnly? Would make sense here.

Reply:
Ping

Reply:
It is not obvious that options contains a mask of flags. Maybe a renaming could help (.h/.cpp)?
`ResolveTrackIdOptions options` -> `ResolveTrackIdFlags flags`